modules/nf-neuro/registration/synthregistration/tests/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+    withName: "REGISTRATION_SYNTHREGISTRATION" {
+        publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+        memory = 20
```

Comment:
Are you sure you need 20Gb of memory ? I don't remember stating anything over 16Gb, if I remember well ...

Reply:
You're right. Don't know why i brought it up.